app/views/questions/index.html.erb

```diff
@@ -26,7 +26,7 @@
       <% end %>
       <form id="questions_searchform" role="search" autocomplete="off" action="/post">
         <div class="input-group">
-          <input tabindex="1" id="questions_searchform_input" type="text" name="title" class="<%= "disabled " if !current_user %>form-control search-query typeahead" qryType="questions" placeholder="type your question">
```

Comment:
Hello @imanishbarnwal  and thanks for working on this...this file was  not part of the issue you have linked...Is this for another issue? if so kindly indicate so...we normally have one pull request per issu. Thanks